File: kattis/inputoutput/leftbeehind/leftbeehind.py
import sys


class leftbeehind:

    def __init__(self, debug: bool = False):
        self.debug = debug

    def dprint(self, *args, **kwargs):
        if self.debug:
            print(*args, **kwargs)

    def generateOutput(self):
        # Take input from user
        self.dprint("-------------------------------------")
        testCaseNumber = 0
        while not testCaseNumber >= 15:
            testCaseNumber += 1
            self.dprint(f"Input the test case {testCaseNumber}.")

            inputNumberOfJars = sys.stdin.readline()
            if not inputNumberOfJars:
                self.dprint("Invalid input.")
                break

            numberOfJars = inputNumberOfJars.strip().split()
            if len(numberOfJars) != 2:
                self.dprint("Input count is not 2.")
                break

            sweetJars = int(numberOfJars[0].strip())
            sourJars = int(numberOfJars[1].strip())
            self.dprint(f"Sweet Jars: {sweetJars} and Sour Jars: {sourJars}")

            match (sweetJars, sourJars):
                case (0, 0):
                    break
                case (sweet, sour) if sweet + sour == 13:
                    print("Never speak again.")
                case (sweet, sour) if sweet < sour:
                    print("Left beehind.")
                case (sweet, sour) if sweet == sour:
                    print("Undecided.")
                case (sweet, sour) if sweet > sour:
                    print("To the convention.")

        self.dprint("-------------------------------------")


if __name__ == "__main__":
    leftbeehind(False).generateOutput()

File: kattis/inputoutput/leftbeehind/leftbeehind.java
package kattis.inputoutput.leftbeehind;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class leftbeehind {

    private final boolean debug;
    private final BufferedReader bufferedReader;
    private StringTokenizer stringTokenizer;

    public leftbeehind(boolean debug) {
        this.debug = debug;
        this.bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    }

    private void dprint(Object... args) {
        if (debug) {
            StringBuilder sb = new StringBuilder();
            for (Object arg : args) {
                sb.append(arg).append(" ");
            }
            System.out.println(sb.toString().trim());
        }
    }

    private String nextToken() throws IOException {
        while (stringTokenizer == null || !stringTokenizer.hasMoreTokens()) {
            String line = bufferedReader.readLine();
            if (line == null)
                return null;
            stringTokenizer = new StringTokenizer(line);
        }
        return stringTokenizer.nextToken();
    }

    public void generateOutput() {
        try {
            // Take input from user
            dprint("-------------------------------------");
            int testCaseNumber = 0;
            while (!(testCaseNumber >= 15)) {
                testCaseNumber += 1;
                dprint("Input the test case " + testCaseNumber + ".");

                String inputSweetJars = nextToken();
                String inputSourJars = nextToken();

                if (inputSweetJars == null || inputSourJars == null || inputSweetJars.trim().isEmpty()
                        || inputSourJars.trim().isEmpty()) {
                    dprint("Invalid input.");
                    break;
                }

                int sweetJars = Integer.parseInt(inputSweetJars.trim());
                int sourJars = Integer.parseInt(inputSourJars.trim());
                dprint("Sweet Jars: " + sweetJars + " and Sour Jars: " + sourJars);

                if (sweetJars == 0 && sourJars == 0) {
                    break;
                } else if (sweetJars + sourJars == 13) {
                    System.out.println("Never speak again.");
                } else if (sweetJars < sourJars) {
                    System.out.println("Left beehind.");
                } else if (sweetJars == sourJars) {
                    System.out.println("Undecided.");
                } else if (sweetJars > sourJars) {
                    System.out.println("To the convention.");
                }

            }
            dprint("-------------------------------------");
        } catch (Exception exception) {
            if (debug)
                exception.printStackTrace();
        }
    }

    public static void main(String[] args) {
        new leftbeehind(false).generateOutput();
    }
}